frontend/server/src/main/java/org/pytorch/serve/wlm/Model.java

```diff
@@ -201,8 +200,9 @@ public void pollBatch(String threadId, long waitTime, Map<String, Job> jobsRepo)
             logger.trace("get first job: {}", Objects.requireNonNull(j).getJobId());
 
             jobsRepo.put(j.getJobId(), j);
-            // describe request job batch size always is 1
-            if (j.getCmd() == WorkerCommands.DESCRIBE) {
+            // batch size always is 1 for describe request job and stream prediction request job
```

Comment:
Not sure I understood this limitation why batch size 1

Reply:
Not really true. We might want a batch of streams as well.It is upto the client.

Reply:
there are two issues when batch size is set >1.

- it breaks current the protocol b/w frontend and backend.
- the latency most likely will be even higher if batch size is larger than 1.